conf/app.routes

```diff
@@ -1,174 +1,15 @@
-GET /                                     controllers.Default.redirect(to = "/cgt-property-disposals/landing-page")
-GET /landing-page                         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.LandingPageController.landingPage()
-GET /sign-in                              uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.LandingPageController.signInPage()
-GET /start                                uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.start()
-GET /we-need-more-details                 uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.weNeedMoreDetails()
-GET /already-subscribed                   uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.onboarding.SubscriptionController.alreadySubscribedWithDifferentGGAccount()
-GET /unsupported-login                    uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.weOnlySupportGG()
-GET /sign-out-and-register-for-gg         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.signOutAndRegisterForGG()
-GET /sign-out-and-log-in                  uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.signOutAndSignIn()
+GET /start/report-pay-capital-gains-tax-uk-property                         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.LandingPageController.landingPage()
+GET /start/report-pay-capital-gains-tax-uk-property/sign-in                 uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.LandingPageController.signInPage()
+GET /create-account/need-more-details                                       uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.weNeedMoreDetails()
+GET /exit/cannot-check-details                                              uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.StartController.weOnlySupportGG()
 
 
-GET /change-gg-account                    uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.onboarding.SubscriptionController.changeGGAccountForSubscription()
+# No mapping in XLS
+GET /                                                                       controllers.Default.redirect(to = "/cgt-property-disposals/landing-page")
```

Comment:
double check these mappings as they don't have any corresponding entries in the xls

Reply:
we don't need this `/` mapping now - it's pointing to the wrong url anyway

Reply:
fixed